fgrpc/pingsrv.go

```diff
@@ -92,11 +94,14 @@ func PingServer(port, cert, key, healthServiceName string, maxConcurrentStreams
 
 // PingClientCall calls the ping service (presumably running as PingServer on
 // the destination). returns the average round trip in seconds.
-func PingClientCall(serverAddr, cacert string, n int, payload string, delay time.Duration) (float64, error) {
+func PingClientCall(serverAddr, cacert string, n int, payload string, payloadSize int, delay time.Duration) (float64, error) {
 	conn, err := Dial(serverAddr, cacert, "") // somehow this never seem to error out, error comes later
 	if err != nil {
 		return -1, err // error already logged
 	}
+	if payloadSize != DefaultPayloadSize {
```

Comment:
change this to `if payloadSize > 0 {`
it will also avoid a crash

but move it out into the caller